src/main/java/bio/terra/service/filedata/azure/AzureSynapsePdao.java

```diff
@@ -351,13 +374,63 @@ public AzureSynapsePdao(
     this.synapseJdbcTemplate = synapseJdbcTemplate;
   }
 
+  /**
+   * Initialize a Synapse database with the given name and encryption key. Note: we need to connect
+   * to the default `master` database to create the new database.
+   */
+  @PostConstruct
+  private void initializeDb() {
+    // If this configuration wasn't set, skip initialization
+    if (azureResourceConfiguration.getSynapse() == null) {
+      return;
+    }
+    boolean initialize = azureResourceConfiguration.getSynapse().isInitialize();
```

Comment:
IMO creating a local variable for `azureResourceConfiguration.getSynapse()` would make this easier to read.

Reply:
Eh, I tend to like fewer variables (especially for 1-level down references